src/class/ConnectError.js

```diff
@@ -51,7 +51,12 @@ export default class ConnectError extends Error {
    */
   constructor(message, code = ERRORS.NOT_DEFINED) {
     super(message);
-    this.stack = new Error().stack;
+    this.name = 'ConnectError';
+    if (Error.captureStackTrace) {
```

Comment:
Does it really need?

Reply:
I think it is the default behaviour for any error

Reply:
in dev tools of chrome we can't see the message of error, that's why we use this solution.
without we can see only `uncautch Error: Error`, but with this we can see `uncautch Error: Error Connect library requires OAuth client id!` for example

Reply:
Ok